docs/plugins/developer/index.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Plugins Developer Guide
+sidebar_label: Developing Plugins
+sidebar_position: 3
+---
+
+import DocCardList from "@theme/DocCardList";
+
+For an overview of Helm Plugin concepts, and how to structure and configure plugins, read the [Plugins Overview](/plugins/overview.md).
```

Comment:
General question here. I assume this document would migrate to the versioned docs and would this absolute path need to be changed?